model/task_lifecycle.go

```diff
@@ -883,7 +883,7 @@ func ClearAndResetStrandedTask(h *host.Host) error {
 			Type:   "system",
 		}
 		if t.IsPartOfDisplay() {
-			return t.DisplayTask.SetResetWhenFinished(detail)
```

Comment:
Is this really a reset, if we construct a new details object on 881?

Reply:
The details object is not used here. I swapped the order to make it clearer